docs/DAB-setup.md

```
@@ -0,0 +1,338 @@
+
+   # Windows PowerShell
+   $env:DAB_HOST_MODE='development'
+   ```
+
+3. **Start DAB** from the project root:
+   ```bash
+   dab start -c ./dab/dab-config.json
+   ```
+
+4. **Verify DAB is running** by visiting the Swagger UI:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)